com.unity.render-pipelines.universal/Shaders/Particles/ParticlesSimpleLit.shader

```diff
@@ -177,7 +177,7 @@ Shader "Universal Render Pipeline/Particles/Simple Lit"
             // Universal Pipeline keywords
             #pragma multi_compile _ _MAIN_LIGHT_SHADOWS _MAIN_LIGHT_SHADOWS_CASCADE _MAIN_LIGHT_SHADOWS_SCREEN
             //#pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
-            #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_SHADOWS
+            //#pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_SHADOWS
```

Comment:
@kaychang-unity is this okay?

Reply:
I think you should keep `_ADDITIONAL_LIGHT_SHADOWS` variant to allow SimpleLit shaders to to generate shadows for additional lights in deferred.

Reply:
awesome that is what I assumed